src/Apps/Collect2/Components/Collection/Header/FeaturedArtists.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, Breakpoint, EntityHeader, Flex, Sans } from "@artsy/palette"
+import React, { useState } from "react"
+import styled from "styled-components"
+
+interface FeaturedArtistsProps {
+  hasMultipleArtists: boolean
+  featuredArtists: object[]
+  breakpointSize: Breakpoint
+}
+
+export const FeaturedArtists: React.FC<FeaturedArtistsProps> = props => {
+  const { featuredArtists, breakpointSize, hasMultipleArtists } = props
+  const artistCount = getArtistCountAtBreakpoint(breakpointSize)
+  const remainingCount = featuredArtists.length - artistCount
+  const truncatedArtists = featuredArtists.slice(0, artistCount)
+  const [showAll, setShowMore] = useState(false)
+  const headlineLabel = "Featured Artist" + (hasMultipleArtists ? "s" : "")
```

Comment:
This code was refactored out of `Header/index`, fixing the performance bottleneck